src/tasks/DailyActivityFlow.py

```
@@ -0,0 +1,1169 @@
+            success_reason="activity_milestone_claimed",
+            verifier=lambda gate, before=target: self._verify_activity_milestone_after_click(before),
+        )
+        self._apply_outcome_to_snapshot(outcome)
+        if outcome.failed:
+            self.snapshot.reward_skip_reason = outcome.failure_reason or "activity_milestone_verification_failed"
+            return False
+        if outcome.skipped:
+            self.snapshot.reward_skip_reason = outcome.skipped_reason or "activity_milestone_gate_rejected"
+            return False
+        self.actions.log_info(
+            "已点击一个顶部阶段奖励入口，"
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)